### PR TITLE
[ADD] hr_birthday_welcome_message

### PR DESCRIPTION
**Pull request by albariera**
This module adds a birthday message to the welcome message on the employee's birthday.

Captura de pantalla de 2022-05-23 16-00-48

### CODE REVIEW

**Review by mtelahun — COMMENTED** [review]
LGTM code review

However, I don't think your tests are being run. In the test log it fails to load the test because it can't import python module freezegun.

**Comment by albariera** [comment]
You are right, thanks @mtelahun :blush:

File: hr_birthday_welcome_message/static/src/js/greeting_message.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    greeting_message.include({
+        init: function(parent, action) {
+            this._super.apply(this, arguments);
+            this.is_birthday = action.is_birthday;
+        },
+
+        _get_birthday_message: function() {
+            return _t(
+                "HAPPY BIRTHDAY! <i class='fa fa-birthday-cake' /> <br/> Enjoy your day"
```
Comment (etobella) [inline hr_birthday_welcome_message/static/src/js/greeting_message.js]: Maybe it is not necessary to be upper case, but just my opinion

**Review by etobella — APPROVED** [review]
Just a small comment, but LGTM in general

File: hr_birthday_welcome_message/static/src/js/greeting_message.js (hunk 1 above)
Comment (albariera) [reply hr_birthday_welcome_message/static/src/js/greeting_message.js]: Changed :)

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**